lib/if.c

```diff
@@ -1140,7 +1143,8 @@ DEFPY_NOSH (interface,
 		 * all interface-level commands are converted to the new
 		 * northbound model.
 		 */
-		ifp = if_lookup_by_name(ifname, vrf_id);
+		if (!ifp)
```

Comment:
Why is this needed if we get the ifp above?

Reply:
I think you re right. I will check it.

Reply:
you are right. removed this.